### PR TITLE
Update definition of scrollSource to allow it to be null

### LINKED ISSUE #11: Fix for https://github.com/WICG/scroll-animations/issues/5 introduced a security bug
https://wicg.github.io/scroll-animations/#dom-scrolltimeline-scrollsource says:

   If this is not specified, the document element of the active document for the current browsing
   context, as determined at the time the ScrollTimeline is constructed, is used.

But the document element of the active document of whatever may not even be same-origin with the code constructing the ScrollTimeline.  So hooking up the ScrollTimeline to it looks to me like a cross-origin information leak.

### PR DESCRIPTION
**Pull request by stephenmcgruer**
This should fix #11, #12, and #13.

### CODE REVIEW

File: Overview.bs [hunk 1]
```diff
@@ -368,8 +370,23 @@ not by wall-clock time, but by the progress of scrolling in a [=scroll container
 <div class="constructors">
 
 :   <dfn constructor for=ScrollTimeline lt="ScrollTimeline(options)">ScrollTimeline(options)</dfn>
-::  Creates a new {{ScrollTimeline}} object. The attributes of the {{ScrollTimeline}} are 
-    initialized with the respective members of the {{options}} dictionary.
+::  Creates a new {{ScrollTimeline}} object using the following procedure:
+
+    1. Let <var>timeline</var> be a new {{ScrollTimeline}} object.
```
Comment (birtles) [inline Overview.bs]: (For what it's worth, I think bikeshed now allows `|timeline|` as a shorthand for specifying variables. I think Tab might also recommend that as being more readable so going forward we should probably prefer that style.)

File: Overview.bs [hunk 2]
```diff
@@ -368,8 +370,23 @@ not by wall-clock time, but by the progress of scrolling in a [=scroll container
...
+
+    1. Let <var>source</var> be the result corresponding to the first matching condition from below.
+
+        : If the <var>scrollSource</var> value of <var>options</var> is non-null,
+        :: Let <var>source</var> be <var>scrollSource</var>
+
+        : Otherwise (<var>scrollSource</var> is null):
+        :: Let <var>source</var> be the {{scrollingElement}} of the {{Document}} <a lt="document associated with a window">associated</a> with the {{Window}} that is the <a>current global object</a>.
```
Comment (birtles) [inline Overview.bs]: This looks good to me although I notice @bzbarsky's comment includes the step, "assert that it's a Window". I imagine that if we have `[Exposed=Window]` on this interface we are ok?

File: Overview.bs (hunk 1 above)
Comment (stephenmcgruer) [reply Overview.bs]: Done.

File: Overview.bs (hunk 2 above)
Comment (stephenmcgruer) [reply Overview.bs]: Added Exposed=Window

**Pull request merged**